scripts/check-generated-bindata-content.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
As per a previous comment, this file should be updating the generated content (i.e. it's intent should be something like `update-bindata.go`). The check for whether the tree is current is performed by `pre-commit.sh` in ci.